File: docs/extensibility/codesnippet/VisualBasic/walkthrough-outlining_9.vb
    Private Shared Function TryGetLevel(ByVal text As String, ByVal startIndex As Integer, ByRef level As Integer) As Boolean
        level = -1
        If text.Length > startIndex + 3 Then
            If Integer.TryParse(text.Substring(startIndex + 1), level) Then
                Return True
            End If
        End If

        Return False
    End Function

File: docs/extensibility/codesnippet/CSharp/walkthrough-outlining_9.cs
    static bool TryGetLevel(string text, int startIndex, out int level)
    {
        level = -1;
        if (text.Length > startIndex + 3)
        {
            if (int.TryParse(text.Substring(startIndex + 1), out level))
                return true;
        }

        return false;
    }